src/App.tsx

```diff
@@ -1,8 +1,14 @@
-import React from 'react';
+import { Component } from 'react';
+import classNames from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goods = [
+type Good = string;
+type State = {
+  selectedGood: Good | '';
```

Comment:
why i can choose only 1 Good?

Reply:
![image](https://user-images.githubusercontent.com/79811705/197065585-024be9b8-b2df-4488-a9e7-afdc6705406a.png)

because it says so in the assignment.
Why are you picking on someone else's code if you don't know what it's about?

Reply:
It's just a question)
Why i can't check your code?
I just want to put some comments for you, it's can be useful for you, no?

Reply:
Do you always go to check everything with people you don't know?

I don't think this is normal, I don't know you, I didn't ask you to look at my code. I have mentors and colleagues who will look at my code and give advice.

Leave my repository.

Reply:
Just chill man, your code is public))